x-pack/plugins/endpoint/server/routes/policy/service.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SearchResponse } from 'elasticsearch';
+import { IScopedClusterClient } from 'kibana/server';
+import { GetHostPolicyResponse, HostPolicyResponse } from '../../../common/types';
+
+export function getESQueryPolicyResponseByHostID(hostID: string, index: string) {
+  return {
+    body: {
+      query: {
+        bool: {
+          must: [
+            {
+              match: {
+                'host.id': hostID,
+              },
+            },
+            {
+              match: {
+                'event.kind': 'policy_response',
```

Comment:
This isn't a valid ECS `event.kind` field per https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html I think we need to change this to `state` or maybe `event` potentially and the other fields in `event.*`. We could set `event.action` to be `policy_response` but I think we should use the `event.dataset` field to filter for `endpoint.policy` or something like that.

But if the only thing that is going to be in this index pattern is policy response messages we probably don't need to add the filter.

Reply:
I can remove it for now, till it is needed.